user.js

```diff
@@ -114,7 +114,7 @@ module.exports = function (common, config, deps) {
       cb = options;
       options = {};
     }
-    cb = cb ?? _.noop;
+    cb = cb || _.noop;
     myToken = newToken;
```

Comment:
@clintonium-119 It looks like this change got merged without making it into the actual `0.52.0` package?